src/machine.rs

```diff
@@ -415,6 +414,10 @@ impl<'mir, 'tcx> Evaluator<'mir, 'tcx> {
         let def_id = frame.instance.def_id();
         def_id.is_local() || self.local_crates.contains(&def_id.krate)
     }
+
+    pub(crate) fn current_span(&self) -> CurrentSpan<'_, 'mir, 'tcx> {
+        CurrentSpan { span: None, machine: self }
+    }
```

Comment:
I think this would better fit in `helpers.rs` (together with the `CurrentSpan` type)

Reply:
I moved it to a separate `impl Evaluator` :shrug: 

I can't figure out how to add it to the `InterpCx` extension trait in that file, because we don't have an `InterpCx` in `Evaluator::memory_read` and `Evaluator::memory_written`.

Reply:
> I moved it to a separate impl Evaluator in that file shrug

Yes, that's what I meant. Thanks. :)